tensorflow/lite/tools/make/targets/linux_makefile.inc

```diff
@@ -4,6 +4,10 @@ ifeq ($(TARGET), linux)
     -fPIC \
     -DGEMMLOWP_ALLOW_SLOW_SCALAR_FALLBACK \
     -pthread
+  CCFLAGS += \
```

Comment:
CXXFLAGS is defined above. Could you explain why CCFLAGS is needed here? Is this specific to your particular building env? As https://stackoverflow.com/questions/5541946/cflags-ccflags-cxxflags-what-exactly-do-these-variables-control indicates, there's no such thing as CCFLAGS in a standard gnu make.

Reply:
The reason is that the `Makefile` is building [`fftsg.c`](https://github.com/tensorflow/tensorflow/blob/af607b76b70c2374c86e6647224516b2234b4022/tensorflow/lite/tools/make/Makefile#L109) using GCC and that will use `CCFLAGS` (not `CXXFLAGS`), see:
https://github.com/tensorflow/tensorflow/blob/af607b76b70c2374c86e6647224516b2234b4022/tensorflow/lite/tools/make/Makefile#L209

Other platform specific makefiles duplicate the `CXXFLAGS` in a similar way:
https://github.com/tensorflow/tensorflow/blob/af607b76b70c2374c86e6647224516b2234b4022/tensorflow/lite/tools/make/targets/aarch64_makefile.inc#L14
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/tools/make/targets/rpi_makefile.inc#L16
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/lite/tools/make/targets/rpi_makefile.inc#L40

So I've tried to stay consistent with what is already being done. Let me know if you'd prefer doing this in a different way.

Reply:
I see. Many thanks for these pointers! Then in this case, I think it might have been a typo as it's supposed to be "CFLAGS" for ".c" files (see
https://github.com/tensorflow/tensorflow/blob/af607b76b70c2374c86e6647224516b2234b4022/tensorflow/lite/tools/make/Makefile#L206-L209) and note the comment itself is not consistent w/ the files to be compiled.

From this perspective, could you change CCFLAGS to CFLAGS in all places just to be consistent w/ standard gnu make? Thanks!

Reply:
I wonder if there might be developer already relying on overriding `CCFLAGS` from the make CLI invocation.

It might be a good idea to still support both.

Reply:
Done. PTAL